lib/src/main/java/graphql/nadel/incremental/NadelIncrementalServiceExecutionResult.kt

```diff
@@ -0,0 +1,14 @@
+package graphql.nadel.incremental
+
+import graphql.nadel.ServiceExecutionResult
+import org.reactivestreams.Publisher
+
+class NadelIncrementalServiceExecutionResult(
```

Comment:
Do we need to introduce this class now? We're not dealing with incremental responses yet. 
This PR is only handling the generation of queries containing `@defer`.

You could change the title of this PR accordingly, so it clearly indicates the stage of the implementation that is being done.